client/v2_2/save_.py

```diff
@@ -146,12 +146,14 @@ def fast(image, directory,
 
   After calling this, the following filesystem will exist:
     directory/
-      config.json  <-- only *.json, the image's config
-      001.tar.gz   <-- the first layer's .tar.gz filesystem delta
-      001.sha256   <-- the sha256 of 1.tar.gz with a "sha256:" prefix.
+      config.json   <-- only *.json, the image's config
+      digest        <-- sha256 digest of the image's manifest
```

Comment:
This change shouldn't affect the code path which produces a tarball that you can `docker load` - that's produced by the `tarball` and `multi_image_tarball` functions, which don't use `fast` or `uncompressed`.

I think the only consumer of this directory is `v2_2_image.FromDisk`, and it looks like it doesn't particularly care about the directory structure, either.

I wish the tests were actually usable, though; it would make me more confident about this change.